.github/workflows/coverage.yml

```diff
@@ -0,0 +1,95 @@
+name: Code coverage
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+      - 'releases/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-22.04']
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          lfs: true
+
+      - name: CMake configure
+        run: cmake -DENABLE_INTEL_MYRIAD_COMMON=OFF -DCMAKE_BUILD_TYPE=Release -DENABLE_FASTER_BUILD=ON -DENABLE_TESTS=ON -DENABLE_COVERAGE=ON -B build
```

Comment:
can we turn off unneeded plugins? E.g. GPU is not used here, GNA also can be omitted.